### DISCUSSION TITLE
Status Column in wfc.Workflow Table

### DISCUSSION THREAD

**Discussion by ghost** [danielgerlag/workflow-core#1294] [Q&A]
Hello Everyone,
I noticed that the status column in the workflow table is set to 2, which indicates that the workflow is suspended right after the completion of the run.

![image](https://github.com/user-attachments/assets/9614add8-2fc4-4268-ab5c-7b69db2278d9)

The item in red is the CompleteTime column which sets the completion of the workflow and the item in blue is the status of the workflow.  You would notice it as 2 which is suspended. 

Is this how it should behave? When would the workflow be marked completed?

**Comment by bainewedlock** [comment]
Status 2 is complete, see here: https://github.com/danielgerlag/workflow-core/blob/d1af4d5d4e58b41d272f91dd9677ffd391e098cc/src/WorkflowCore/Models/WorkflowInstance.cs#L41
